guide/02-api-overview/release-notes.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>needs to add this:</p><p>https://github.com/ArcGIS/geosaurus/pull/6035</p><p>https://github.com/ArcGIS/geosaurus/pull/6031</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
- [x] - done!